client/mysqlbinlog.cc

```diff
@@ -2937,7 +2938,21 @@ static Exit_status dump_local_log_entries(PRINT_EVENT_INFO *print_event_info,
               llstr(old_off,llbuff));
         goto err;
       }
-      // file->error == 0 means EOF, that's OK, we break in this case
+      // else file->error == 0 means EOF, that's OK, we break in this case
+
+      /*
+        If we're not in raw mode, then emit a warning in the event that we
+        finished processing input before reaching the boundary indicated by
+        --stop-position.
+      */
+      if (!opt_raw_mode &&  // --stop-position is ignored in raw mode
```

Comment:
I don't think we need to check for raw mode, as I don't think we can actually be in this function when that is set (as its path needs a remote server, which goes to `dump_remote_log_entries()`).